workers/ohsome_quality_analyst/api.py

```diff
@@ -88,38 +80,65 @@ async def check_geom_size(geom):
 
 @app.get("/indicator/{name}")
 async def get_indicator(
-    name: str,
-    request: Request,
-    layerName: str,
+    name: IndicatorEnum,
+    layerName: LayerEnum,
     bpolys: Optional[str] = None,
-    dataset: Optional[str] = None,
+    dataset: Optional[DatasetEnum] = None,
     featureId: Optional[str] = None,
-    fidField: Optional[str] = None,
+    fidField: Optional[FidFieldEnum] = None,
 ):
-    url = request.url._url
```

Comment:
Do we want to include the request URL in the response?

Reply:
After internal discussion the request URL will be removed from the API response until it is requested by endusers.